pkg/apiserver/registry/stats/antreaclusternetworkpolicystats/rest_test.go

```diff
@@ -256,3 +264,70 @@ func TestRESTList(t *testing.T) {
 		})
 	}
 }
+
+func TestRESTConvertToTable(t *testing.T) {
+	stats := &statsv1alpha1.AntreaClusterNetworkPolicyStats{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace:         "foo",
+			Name:              "bar",
+			CreationTimestamp: metav1.Time{Time: time.Now()},
+		},
+		TrafficStats: statsv1alpha1.TrafficStats{
+			Packets:  10,
+			Bytes:    2000,
+			Sessions: 5,
+		},
+	}
+	tests := []struct {
+		name          string
+		object        runtime.Object
+		expectedTable *metav1.Table
+	}{
+		{
+			name:   "one object",
+			object: stats,
+			expectedTable: &metav1.Table{
+				ColumnDefinitions: []metav1.TableColumnDefinition{
```

Comment:
is it useful to redefine `ColumnDefinitions` here, or could we just use a shared global variable with the `ConvertToTable` function?

Reply:
extracted a variable as you suggested